packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart

```diff
@@ -410,6 +414,54 @@ void main() {
       }
     }
   },
+  
```

Comment:
A bunch of trailing whitespace analysis failures: 
```
▌10:29:03▐ STARTING ANALYSIS
▌10:29:03▐ runtimeType in toString...
▌10:29:03▐ Unexpected binaries...
▌10:29:03▐ RUNNING: cd .; git ls-files -z
▌10:29:03▐ ELAPSED TIME: 0.032s for git ls-files -z in .
▌10:29:03▐ Trailing spaces...
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
/b/s/w/ir/k/flutter sdk/packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart:417: trailing U+0020 space character
/b/s/w/ir/k/flutter sdk/packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart:432: trailing U+0020 space character
/b/s/w/ir/k/flutter sdk/packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart:443: trailing U+0020 space character
/b/s/w/ir/k/flutter sdk/packages/flutter_tools/test/integration.shard/test_data/gen_l10n_project.dart:454: trailing U+0020 space character
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
```

Reply:
how did you run this analysis?

Reply:
Run `flutter analyze --flutter-repo`. If that doesn't work, run `flutter analyze` in `flutter/packages/flutter_tools`.